commit.go

```diff
@@ -40,6 +40,68 @@ func (c *Commit) RawMessage() string {
 	return ret
 }
 
+func (c *Commit) RawHeader() string {
```

Comment:
one more thing i forgot, can the public functions have a docstring? most of the contents can be copied from the C docs.

Reply:
Fixed